source/packages/services/installer/src/commands/modules/service/certificateVendor.ts

```diff
@@ -66,6 +66,49 @@ export class CertificateVendorInstaller implements ServiceModule {
         default: false,
         askAnswered: true,
       },
+      {
+        message: 'Will you use ACMPCA to issue the certificate ?',
```

Comment:
Move the ACMPCA questions as the first set of questions.
Add additional question for multi account and multi region support (see provisioning installer for list of prompts)
There will be requirements for multi ACMPCA certificates (see provisioning installer `Create or modify ACM PCA CA alias list ?`)

Reply:
I think we have 2 choices for certificateVendor, 1)Put the pre-created certificate on S3 and use that certificate, 2) use CSR from the device.  For the S3 case, we just use that certificate, we do not have to change anything. to use or not use ACMPCA is only effective for 2) CSR case. Here ACMPCA feature has dependency to the question of 'Will you be requesting certificates using a CSR?' Thats why we put ACMPCA questions after CSR question. I will add multi-account support related question and multi-region support related questions.

Reply:
added change. Please review. https://github.com/aws/aws-connected-device-framework/pull/93/commits/6c21c7501abb92e72a103530cd7037d45250e144

Reply:
Based on the conversation, I have added the CA alias list. Please check.